agreements/project-contribution/terms-and-conditions.md

```diff
@@ -1,4 +1,293 @@
+
+**Contributor Agreements:**
+Projects must require contributors to comply with a Developer Certificate of Origin (DCO) or a Contributor License Agreement (CLA) as outlined in the [IP Policy][].
+
+**Sponsor-owned Assets:**
+The Sponsor owns all domains, trademarks, and other intellectual property necessary to manage the Project.
+Registration, renewal, and other maintenance of these assets are the responsibility of the Sponsor and must be funded using Sponsor-managed funds.
+Contributions or donations held in support of these purposes must be transferred to the Sponsor’s restricted fund.
+
+**Trademark Collaboration:** Project Signatories may develop new identifiers, such as logos, to support the Project’s identity.
+These identifiers must meet the Sponsor’s quality and trademark standards and be approved before use.
+Once approved, these identifiers will be added to the Sponsor’s [Trademark List][trademarks], with all goodwill benefiting the Sponsor.
+
+## Financial Management
+
+The Sponsor will manage funds on behalf of the Project in compliance with its tax-exempt status (501(c)(6)).
+All funds will be handled according to the Sponsor’s financial policies and procedures.
+The Sponsor retains the right to approve or deny expenses to ensure compliance with applicable laws and the Project’s purposes.
+This authority is detailed further in the [Fiscal Sponsorship Agreement][fs-landing].
+
+### Restricted Fund Management
+
+If the project opts to manage its finances through the Sponsor, all contributions will be placed in a restricted fund established for the Project, as detailed in the [Fiscal Sponsorship Agreement][fs-landing], to manage contributions and expenses related to the Project Purposes.
+
+#### Use of Funds
+
+Projects must use Sponsor-managed funds received for stated activities only and maintain a public and transparent ledger for the disbursement of those funds, as described in the [Fiscal Sponsorship Agreement][fs-landing].
+
+#### Administrative Fees
+
+The Sponsor may debit administrative fees from the Project’s restricted fund as specified in the [Fiscal Sponsorship Agreement][fs-landing].
+Details about fee structures and any adjustments are available upon request.
+
+#### Platform
+
+The Sponsor currently uses the Open Collective platform (the “Platform”) to manage restricted funds for hosted projects.
+Upon execution of the Fiscal Sponsorship Agreement, a new Collective (or a Collective Project) will be created for the restricted fund.
+
+The Sponsor may switch to another platform with similar capabilities if necessary, with advance notice to the Project.
+
+#### Expenses and Disbursements
+
+Projects must submit expense reports (with receipts and documentation) through the Platform.
+
+All expenses must be approved by the Sponsor to ensure they are valid and in alignment with the Sponsor's tax-exempt purposes under the terms of this agreement before disbursement.
```

Comment:
At the moment, OC hosted collective administrators can approve their own expenses. We should probably need to add some language saying something like "Hosted collectives admins must refrain from approving their own expenses when submitting them via the platform."

Reply:
Or we state that approved expenses are subject to Sponsor review/oversight?

Reply:
suggest?

Reply:
```suggestion
All expenses must be approved by the Sponsor to ensure they are valid and in alignment with the Sponsor's tax-exempt purposes under the terms of this agreement before disbursement. Project restricted funds hosted at the OpenCollective, expenses can be approved by Project Leaders but the Sponsor reserves the right to deny expenses not in alignment with the Sponsor's tax-exempt purposes under the terms of this agreement post disbursement.
```

Reply:
This was helpful, Ken!

I chewed on it for a bit, and looked back at some others, and have suggested something else. 

@kenfinnigan and @cealsair ... have a look @ new section.